### PR TITLE
API Subscription Flow Implementation

### PR DESCRIPTION
**Pull request by DivyaPremanantha**

### CODE REVIEW

File: src/controllers/myAPIsController.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+const fs = require('fs');
+const { renderGivenTemplate, loadLayoutFromAPI, renderTemplate } = require('../utils/util');
+const config = require(process.cwd() + '/config.json');
+const constants = require('../utils/constants');
+const adminDao = require('../dao/admin');
+const apiMetadataService = require('../services/apiMetadataService');
+const util = require('../utils/util');
+
+const loadMyAPIs = async (req, res) => {
+    const orgName = req.originalUrl.split("/")[1];
```
Comment (SachiniSiriwardene) [inline src/controllers/myAPIsController.js:29]: lets use req.params.orgName

File: src/controllers/myAPIsController.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                    applicationId: sub.applicationInfo.applicationId,
+                    throttlingTier: sub.throttlingPolicy,
+                    appStatus: sub.status,
+                });
+
+            }
+        }
+    }
+
+    if (req.query.apiId) {
```
Comment (SachiniSiriwardene) [inline src/controllers/myAPIsController.js:59]: lets add a comment explaining the logic in short...

Comment (DivyaPremanantha) [reply src/controllers/myAPIsController.js:60]: Added https://github.com/wso2/api-developer-portal-core/pull/71/commits/13b2241cd5dc9788a322993613ffd06c785c10a2

File: src/dao/admin.js [hunk 3]
```diff
@@ -56,6 +56,22 @@ const getOrganization = async (param) => {
     }
 };
 
+const getOrgId = async (orgName) => {
+        try {
+            const organization = await Organization.findOne({ where: { ORG_NAME: orgName } });
+            if (!organization) {
+                console.log('orgName', orgName);
```
[the remark this answers is not in the document]
Comment (DivyaPremanantha) [reply src/dao/admin.js]: Removed in: https://github.com/wso2/api-developer-portal-core/pull/71/commits/13b2241cd5dc9788a322993613ffd06c785c10a2

File: src/controllers/myAPIsController.js (hunk 1 above)
Comment (DivyaPremanantha) [reply src/controllers/myAPIsController.js]: Fixed https://github.com/wso2/api-developer-portal-core/pull/71/commits/13b2241cd5dc9788a322993613ffd06c785c10a2

**Review by SachiniSiriwardene — APPROVED** [review]

**Pull request merged**